hedy.py

```diff
@@ -1169,6 +1178,50 @@ def assign(self, args):
             values = args[1:]
             return parameter + " = [" + ", ".join(values) + "]"
 
+@hedy_transpiler(level=12)
+class ConvertToPython_12(ConvertToPython_11):
+    def process_comparison(self, args, operator):
+
+        # we are generating an fstring now
+        arg0 = process_variable_for_fstring_padded(args[0], self.lookup)
+        arg1 = process_variable_for_fstring_padded(args[1], self.lookup)
+
+        # zfill(100) leftpads variable to length 100 with zeroes (hence the z fill)
```

Comment:
The comment about `zfill` does not make sense without knowing the implementation of `process_variable_for_fstring_padded`